ProblemTwo/trivia.js

```diff
@@ -0,0 +1,40 @@
+//document.addEventListener("DOMContentLoaded",()=>{
+//   let main = document.querySelector(".main")
+//       axios.get("http://localhost:4001").then(response=>{
+//           let data = response.data.results 
+//           debugger
+//           data.forEach(el=>{
+//             let question = document.querySelector("p")
+//             let button = document.querySelector("button")
+//             let answers = document.querySelector("select")
+//             question.innerText = el.results.question
+//             answers.innerText
+//           })
+//         })
+// })
+
+document.addEventListener("DOMContentLoaded", () => {
+  let submitButton = document.querySelector("#submitButton")
+  let p = document.querySelector("#p")
+  let select = document.querySelector("#select")
+  let quizData
+
+
+  const questionSelect = (data) => {
+    // let wrongOptions = data.incorrect_answers
```

Comment:
would love to see you finish this. It's very similar to the Unit 2 exam - which you've completed and know how to do :)